timelock-server/src/testCommon/java/com/palantir/atlasdb/timelock/util/ExceptionMatchers.java

```diff
@@ -17,16 +17,15 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import feign.RetryableException;
+
 public final class ExceptionMatchers {
 
     private ExceptionMatchers() { }
 
     public static void isRetryableExceptionWhereLeaderCannotBeFound(Throwable throwable) {
         assertThat(throwable)
-                .hasMessageContaining("method invoked on a non-leader");
-
-        // We shade Feign, so we can't rely on our client's RetryableException exactly matching ours.
-        assertThat(throwable.getClass().getName())
-                .contains("RetryableException");
+                .hasMessageContaining("receiving QosException.RetryOther")
+                .isInstanceOf(RetryableException.class);
```

Comment:
do we want to look at the cause and map that to a `QosException.RetryOther`?

Reply:
I'll see what I can do about this

Reply:
(yep, the root cause can be attributed to one of those)